conf/entrypoint.sh

```diff
@@ -1,7 +1,11 @@
 #!/usr/bin/env bash
 # It will generally take about 40 seconds for elasticsearch and couchdb to be ready to receive connections
 echo 'Scheduling setup scripts to run in 40 seconds...'
-sleep 40 && /usr/src/app/conf/initcouch.sh  2>&1 && /usr/src/app/utils/elasticsearch.sh couchadmin test 2>&1 &
+sleep 40
```

Comment:
Here, for instance, they try to postpone the script execution, but it doesn't work. Without `&&` it works as expected and really stops the script from the execution. I believe that was the expected behavior. Opinions?

Reply:
I think your change is correct. Can we remove the part where it runs the scripts the second time? Is it possible they just included the second one because the first one always failed because it wasn't actually waiting the 40 seconds like it was supposed to?